### DISCUSSION TITLE
Waterfox G5??

### DISCUSSION THREAD

**Discussion by RJVB** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#59] [General]
Hi,

I've noticed Waterfox G5 is out and at 5.0.2 or so. Any plans to provide AppImages for this new version, now that I presume G4 is on life support at best?

**Comment by hawkeye116477** [comment]
It's planned. I started working on it few days ago, but have problems with building => https://build.opensuse.org/package/live_build_log/home:hawkeye116477:waterfox/waterfox-g-appimage/CentOS_7/x86_64. When I figured out that, it will be.

**Comment by RJVB** [comment]
On Thursday November 10 2022 04:16:33 hawkeye116477 wrote:
>It's planned. I started working on it few days ago, but has problems with building => https://build.opensuse.org/package/live_build_log/home:hawkeye116477:waterfox/waterfox-g-appimage/CentOS_7/x86_64. When I figured out that, it will be. 

Good to know, thanks!

Is this your error?

` UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 7275: ordinal not in range(128)`

I remember having seen this before, I *think* after starting to use a new python version (> 2.7?). That was years ago, I probably won't be able to remember how I solved it exactly but I seem to remember it had to do with setting another default encoding. It's probably going to be easier for you to patch out the offending characters

**Reply by hawkeye116477** [reply]
Yeah, that's it. Same problem I've on CentOS 7 and Ubuntu 18.04, but doesn't appears on newer Ubuntu versions. 
Looks like that setting `LC_ALL=C.UTF-8` is enough to fix that.

**Comment by RJVB** [comment]
Sounds like it should do the trick, indeed!

**Comment by hawkeye116477** [comment]
AppImage for latest Waterfox G.x is now available at https://download.opensuse.org/repositories/home:/hawkeye116477:/waterfox/AppImage/waterfox-g-latest-x86_64.AppImage.mirrorlist.

**Comment by RJVB** [comment]
Great, thanks!

I see it now identifies as "waterfox-g" in the AppRun script. Is that you, or an upstream change?

**Reply by hawkeye116477** [reply]
It's my change. In upstream it was and still is just `waterfox` in case of G.x and Classic, but if both versions are installed, then that can cause issues, so that's why I'm renaming.

**Comment by RJVB** [comment]
Sure, I was just surprised that it went from waterfox-g4 to waterfox-g; I would have expected waterfox-g5 .

**Reply by hawkeye116477** [reply]
It was made for easier future transitions and little less work for future versions :smile:.

**Comment by RJVB** [comment]
Fair enough!

I see a new bunch of warning about failure to open /dev/dri/renderDXXX because of permissions, including the single renderD128 device I have. I am member of the `video` group that has rw perms, so any idea what this is about?
